src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -505,13 +511,13 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
 
             user_agent = http_user_agent(user_agent)
 
-            if is_safetensors_available():
+            if use_safetensors in {None, True}:
```

Comment:
```suggestion
            if use_safetensors or use_safetensors is None:
```

Reply:
actually I would like to avoid the extra call to the Hub if `safetensors` are not installed. Can we make sure that we still check for whether safetensors are installed?